frontend/src/pages/Landing/Filter.js

```diff
@@ -76,6 +76,7 @@ function Filter({ applyFilters, forMyAlerts }) {
     // https://reactjs.org/docs/events.html#detecting-focus-entering-and-leaving
     // e.relatedTarget can be null when focus changes within the menu (when using VoiceOver)
 
+    // const isCalendarControl = e.target.classList.contains('CalendarDay') || e.target.classList.contains('DayPickerNavigation_button');
```

Comment:
Not sure if this line is supposed to be commented out? If its not needed we can probably remove it.